app/forms/supply_teacher_form.rb

```diff
@@ -0,0 +1,34 @@
+class SupplyTeacherForm < Form
```

Comment:
What do you think about namespacing this form under `Journeys:AdditionalPaymentsForTeaching`?

Reply:
i like it, updated 👍